src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String color = ColorSupplier.getRandomColor();
```

Comment:
The method 'getRandomColor' should be called on an instance of 'ColorSupplier', not statically. You should create an instance of 'ColorSupplier' and call the method on that instance.